### ISSUE TITLE
Suggestion: add installation instructions on readme

### ISSUE THREAD

**Issue by Jaxelr** [kevcunnane/ssmskeymap#3]
Although i'm aware that the installations instructions can be found elsewhere, the readme included as a preview makes no mention of the installation process. I think it would be helpful for newbies to have them.

**Comment by sjc01234** [comment]
I am also totally in the dark of where the "contribution's list" is to view what shortcuts are made available by this extension for ADS.

**Comment by coffeebeantraining** [comment]
Agreed, at first i tried installing like everyone else and got redirected to the github repo.
then i discovered this is the way to install:
![snap_screen_20221118185251](https://user-images.githubusercontent.com/59173960/202830962-78180268-577c-4c93-bad5-addf5ff226a5.png)
